config/application.rb

```diff
@@ -57,5 +57,6 @@ class Application < Rails::Application
     end
 
     config.rncs_sources = config_for(:rncs_sources)
+    config.rncs_sources_path = ::File.join(Rails.configuration.rncs_sources['path_prefix'], Rails.configuration.rncs_sources['ftp_path_prefix'])
```

Comment:
Le naming est pas top ici j'ai l'impression, j'ai du mal à comprendre aisément pourquoi join path_prefix avec ftp_path_prefix

Reply:
`path_prefix` c'est le dossier dans lequel je mets les fichiers synchronisés : wget télécharge le contenu du FTP et le range dans ce dossier, par exemple `/path/to/sources/`.

`ftp_path_prefix` représente l'arborescence parente sur le FTP source qui sera répliqué localement lorsque wget télécharge les fichiers. Par exemple, si je fais `wget -r ... ftps://mon_ftp.fr/path/to/file` la commande télécharge le fichier ici `./mon_ftp.fr/path/to/file`. 

J'ai externalisé des variables pour pas avoir à trainer toute l'arborescence qui se répète à chaque fois, après téléchargement les fichiers sont rangés dans `<path_prefix>/<ftp_path_prefix>/tc/flux/<year>/<month>/<day>...` et ce qui m'intéresse pour les traitements c'est la fin du path (TC ou TITMC, stock ou flux, date des MAJ, ...). Le reste ça bouge pas.

Le fait d'avoir externalisé les dossiers parents où ranger les fichiers ça permet aussi de différencier les dossiers de rangement des fichiers sources pour les différents environnements. Par exemple, pour l'environnement de test : 

```yml
path_prefix: ./spec/fixtures
ftp_path_prefix: ''
```

Pour les tests, le code ira donc chercher les fichiers dans `./spec/fixtures/tc/stock/...`

Reply:
Je suis 100% en accord avec ta démarche, mon point c'est que ces explications que tu vient de me donner ne se trouvent pas dans le code, je pense qu'il faudrait, quitte à devoir écrire un fat commentaire

Reply:
> Sinon le fait que depuis le code, tu puisses remonter au commit, à la PR et à la description de la PR suffit si la description de celle-ci permet de comprendre ce qu'il se passe.

Github ça sert pas que pour les review, faut pas oublier que c'est très utile pour retrouver du contexte, et surtout le "pourquoi" d'une ligne de code. J'ai pas trop envie d'écrire un "fat commentaire" dans le code.